includes/admin/class-wc-rest-payments-disputes-controller.php

```diff
@@ -60,16 +60,6 @@ public function register_routes() {
 			]
 		);
 
-		register_rest_route(
```

Comment:
This change will likely impact code in `woocommerce-payments/client/card-readers/settings/file-upload.js`. I hope you've already tested it to be working fine.

Reply:
I used the same path as before ( `payments/file` ), so we should be safe